pkg/cmd/objects/browse/browse.go

```diff
@@ -43,8 +43,17 @@ func NewBrowseCmd(f *cmdutil.Factory) *cobra.Command {
 			This command browse the objects of the specified index.
 		`),
 		Example: heredoc.Doc(`
-			# Browse the objects from the "TEST_PRODUCTS_1" index
-			$ algolia objects browse TEST_PRODUCTS_1
+			# Browse the objects from the "BOOKS" index
+			$ algolia objects browse BOOKS
+
+			# Browse the objects from the "BOOKS" index and select which attributes to retrieve
+			$ algolia objects browse BOOKS --attributesToRetrieve author,title,description
+
+			# Browse the objects from the "BOOKS" index with filters
+			$ algolia search BOOKS --filters "'(category:Book OR category:Ebook) AND _tags:published'"
+
+			# Browse the objects from the "BOOKS" index with custom pagination
+			$ algolia search BOOKS --hitsPerPage 10 --page 3
```

Comment:
```suggestion
			$ algolia search BOOKS --hitsPerPage 10 --page 3
```

I think `browse` doesn't use pagination.

Reply:
You're right, I tried and it doesn't work. However it's written in the doc: https://www.algolia.com/doc/tools/cli/commands/algolia-objects/#algolia-objects-browse-pagination

Should we remove these options from the doc?